src/compiler/sourcemap.ts

```diff
@@ -4,6 +4,65 @@ namespace ts {
         extendedDiagnostics?: boolean;
     }
 
+    const base64CharCodeForValue = [

```

Comment:
It doesn't look like this was much of a win compared to the compact function it replaced?

If we do decide to keep it, it seems like there need to be entries for the digits?

Reply:
Yeah, I'd say keep the old function.

Reply:
oops

Reply:
Yeah, the difference there is pretty small, reverting to scope the change.